src/dbus/client/thread_api_dbus.hpp

```diff
@@ -231,6 +234,42 @@ class ThreadApiDBus
      */
     ClientError RemoveExternalRoute(const Ip6Prefix &aPrefix);
 
+    /**
+     * This method starts the commissioner.
+     *
+     * @param[in]   aCommissionerStateHandler   The commissioner state handler.
+     * @param[in]   aJoinerEventHandler         The joiner event handler.
+     *
+     * @retval ERROR_NONE successfully performed the dbus function call
```

Comment:
nit: prefer complete sentences in the description (here and below)
```suggestion
     * @retval ERROR_NONE Successfully performed the dbus function call.
```